etc/ssdlc.md

```diff
@@ -0,0 +1,48 @@
+# MongoDB C Driver SSDLC Compliance Report
+
+<!--
+This document is intended for distribution with release archives of the MongoDB
+C Driver. It wil be copied into the release archive as ssdlc_compliance_report.md
+-->
+
+## Release Creator
+
+For information on the release creator, refer to the
+[C/C++ Release Info Spreadsheet](https://docs.google.com/spreadsheets/d/1yHfGmDnbA5-Qt8FX4tKWC5xk9AhzYZx1SKF4AD36ecY)
+(internal link).
+
+## Process Document
+
+<!-- DRIVERS-2892: replace with link to public-facing document once available. -->
+Not yet available.
+
+## Third-Party Dependencies and Vulnerabilities
+
+The tracking of security information for bundled third-party dependencies is
+performed using Snyk. Refer to the `etc/third_party_vulnerabilities.md` file
+within the repository or the release archive for vulnerability information known
+at the time of a release. Full bundled dependency information is available in
+the `cyclonedx.sbom.json` file within the release archive.
+
+## Static Analysis Findings
+
+Refer to the
+[SSDLC Static Analysis Reports](https://drive.google.com/drive/folders/17bjBnQ3mhEXvs6IK1rrTphJr0CUO2qZh)
```

Comment:
May want to move this into the existing [C Driver](https://drive.google.com/drive/folders/1Q6U43xy8M_JF-jKVfz26uS1oGbAI-fBw) folder, e.g. as done for the [CXX Driver](https://drive.google.com/drive/folders/1Yq9wzMQ4O-7dvIeLp-dik-o9b5A4rzhs).

Reply:
The release instructions defer to the `Creating SSDLC static analysis reports` doc, which instructs placing those reports within the `SSDLC Static Analysis Reports` sub-folder (linked here). It may be preferable to inline the content of `Create SSDLC static analysis reports` into `releasing.rst` (it's not a big document) to get it into version control.

The folder structure may need some cleaning-up, because now we have:

```
- DBX Teams > C/C++ Team
  - C Driver/
    - Archive/
  - C++ Driver/
    - Archive/
    - SSDLC Reports/
  - libmongocrypt/
    - Archive/
  - SSDLC/
    - SSDLC Static Analysis Reports/
```

@kevinAlbs opinions?

Aside: I really really really don't like using "just a bunch of files in a shared folder" for things we want to track carefully. This feels like pre-CVS version control and I'm 90% sure this will come back to bite everyone.

Reply:
> I really really really don't like using "just a bunch of files in a shared folder" for things we want to track carefully.

Ultimately, all we need to do is ensure the reports are stored and remain accessible on-demand (by customer request, which is the responsibility of the product manager(s) fulfilling that request). We do not need to actively maintain the contents once they've been archived. The actual final directory to which reports must be uploaded [isn't even under the DBX C/C++ path](https://drive.google.com/drive/folders/1ivg27K48Me720nrxXwddis6QNs5SlQTP) anyways, so how we organize the master/draft files is not really a serious concern.

The advice to use existing per-repo folders was to preserve separation of concerns. The parent `SSDLC` folder was created very early in SSDLC policy development stages before the current requirements were fully understood.

Reply:
`SSDLC Static Analysis Reports` was kept out of the per-project directories to support storing reports for all of C, C++, and libmongocrypt. I am not opposed to changing. Having separate `SSDLC Reports` directories for C and libmongocrypt for the static analysis reports may help organize.